wgsl/index.bs

```diff
@@ -9932,10 +9944,11 @@ The following algorithm describes how to compute these tags for a given function
              set the [=call site tag=] to [=CallSiteRequiredToBeUniform.S=], and set its [=witness set=] to *W*.
         * For each [=param_i=] in *R.S*, if its corresponding [=parameter tag=] has not been updated since initialization, then
              set that tag to [=ParameterRequiredToBeUniform.S=], and set its [=witness set=] to *W*.
+* Mark all the nodes as unvisited.
 * If [=Value_return=] exists, let *VR* be the set of nodes reachable from [=Value_return=].
     * If *VR* includes [=MayBeNonUniform=], then set the [=function tag=] to [=ReturnValueMayBeNonUniform=].
-    * For each [=param_i=] in *VR*, if the corresponding [=parameter tag=] has not been updated since initialization,
-         then set it to [=ParameterRequiredToBeUniformForReturnValue=].
+    * For each [=param_i=] in *VR*, if the corresponding [=parameter return tag=] has not been updated since initialization,
```

Comment:
We don't need the condition that it has not been updated since initialization.
It's ok to set it multiple times because there is only one bad value.

I only needed the condition when this was updating the parameter tag that needed to keep the severity information (and witness sets).